src/app/stack/card-details/card-details.component.html

```diff
@@ -1,24 +1,35 @@
 <div class="col-xs-12 analytics-card-details">
-    <h2 [innerText]="details?.title"></h2>
-    <p [innerText]="details?.titleDescription"></p>
-    <div *ngIf="details?.reportInformations && details?.reportInformations.length > 1">
+
+    <div class="col-xs-10 pl-0 ml-0">
+        <h2 [innerText]="details?.title"></h2>
+        <p [innerText]="details?.titleDescription"></p>
+    </div>
+    <div class="col-xs-2">
+        <a href="/" title="Sign up with Snyk">
+            <span class="power">Powered By </span>
+            <img src="../../.././assets/icon/snky-logo.png" alt="snyk-logo" class="logo" />
```

Comment:
Probably we need to check with Snyk regarding which logo to use. Also should we download it from their servers?

Reply:
For Snyk Logo I am using their official press-kit png files which they have published on their website : https://snyk.io/press-kit/
Yes we can, download it from their servers -> I would need more information on this.